server/services/socket.js

```diff
@@ -48,7 +56,10 @@ exports.emitWinner = player => io.emit("winner", player);
 exports.emitEndGame = () => io.emit("end");
 
 // Ask the current player for a word and a card
-const promptCurrentPlayer = currentPlayer => sockets.find(socket => socket.handshake.session.user === currentPlayer.username).emit("play word and card");
+const promptCurrentPlayer = currentPlayer => {
+    let current = sockets.find(socket => socket.handshake.session.user === currentPlayer.username)
+    if (current) current.emit("play word and card");
```

Comment:
Why did this get split into two? Is there an error somewhere else that we need to find to prevent this from crashing?

Reply:
Undefined still exists in the sockets array, the thing I said in the group chat last night